plugins/woocommerce-admin/client/homescreen/mobile-app-modal/components/MobileAppInstallationInfo.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import MobileQRCode from '../illustrations/woo-mobile-download-qrcode.svg';
+
+export const MobileAppInstallationInfo = () => {
+	return (
+		<div>
+			<img
+				src={ MobileQRCode }
+				alt={ __( 'Download WooCommerce mobile app', 'woocommerce' ) }
+				width="100"
```

Comment:
I wonder if this could be a little bit bigger? It looks pretty small on my desktop display:

![modal-qr](https://github.com/woocommerce/woocommerce/assets/916023/41c7fdbd-dfd2-40da-915d-840c30c9bd7a)

Reply:
Yea great point, the QR code for Jetpack mobile https://apps.wordpress.com/mobile/ is also pretty big (18.3 rem).

I updated the QR code size to 150x150 px in https://github.com/woocommerce/woocommerce/pull/40559/commits/780c5216b583a5b31548b9f00ad33c7b410f5bb1 (screenshots updated in the PR description).